liquibase-standard/src/test/groovy/liquibase/changelog/DatabaseChangeLogTest.groovy

```diff
@@ -432,6 +433,88 @@ create view sql_view as select * from sql_table;'''
         bufferLog.getLogAsString(Level.WARNING) == ""
     }
 
+    def "start with classpath: and hidden"() {
+        when:
+        def resourceAccessor = new MockResourceAccessor([
+                "classpath:com/example/.hiddenfile": test1Xml,
+        ])
+
+        // TODO: Not use deprecated
+        BufferedLogService bufferLog = new BufferedLogService()
+        def rootChangeLog = new DatabaseChangeLog("com/example/root.xml")
+
+        Scope.child([
+                (Scope.Attr.logService.name())                                                     : bufferLog,
+                (ChangeLogParserConfiguration.ON_MISSING_INCLUDE_CHANGELOG.getKey())               : ChangeLogParserConfiguration.MissingIncludeConfiguration.WARN,
+        ], new Scope.ScopedRunner() {
+            @Override
+            void run() throws Exception {
+                rootChangeLog.include("classpath:com/example/.hiddenfile", false, true, resourceAccessor, new ContextExpression("context1"), new Labels("label1"), false, null, DatabaseChangeLog.OnUnknownFileFormat.WARN, new ModifyChangeSets(null, null))
+            }
+        })
+
+        def changeSets= rootChangeLog.changeSets
+
+        then:
+        changeSets.isEmpty() == true
+        bufferLog.getLogAsString(Level.WARNING) == ""
+    }
+
+    def "start with classpath: and cvs"() {
```

Comment:
What are you trying to test with this scenario? Whether a file with classpath prefix and extension is a `csv`. Because I can see in the mock that we are relating this path( which is not a CSV file extension) to a `XML` file. I might not be getting the purpose of this scenario.